documentation/definition-of-done.md

```diff
@@ -1,20 +1,16 @@
-# Definition of Done:
+# Definition of Done
```

Comment:
I always forget what this document represents.  Worth adding a sentence or two to introduce it?

Reply:
That is a great question. **Definition of Done** is an agile term and I am not sure we are actually following what agile suggests to include 😨.

Reply:
A quick google search takes me to https://www.leadingagile.com/2017/02/definition-of-done/ .  It's interesting that it notes that the DoD is different at different stages.  The description in the "Feature" section sounds quite similar (to me) to what we're aiming for.

Reply:
Also, it reminds me that "completing the acceptance criteria" should probably be on the list!  :laughing: